_posts/2014-08-01-accounts.markdown

```diff
@@ -22,9 +22,17 @@ account creation.
 | `user:first_name`             | String            |           |          |         | First name of the primary payroll administrator
 | `user:last_name`              | String            |           |          |         | Last name of the primary payroll administrator
 | `user:email`                  | String            |           |          |         | Email address for primary payroll administrator. Will also be used as their login.
+| `user:phone`                  | String            |           |      Yes |         | Phone number for primary payroll administrator.
 | `company`                     | Object            |           |          |         | Information for the new company
 | `company:name`                | String            |           |          |         | Company name. Must be at least two characters long.
+| `company:trade_name`          | String            |           |      Yes |         | Company legal name.
+| `company:ein`                 | String            |           |      Yes |         | Company's EIN (aka FEIN)
 | `company:states`              | Array             |           |          |         | List of states (two letter abbreviations) in which the company operates.
+| `partner`                     | Object            |           |      Yes |         | Partnership Information
```

Comment:
`Partner Information` should be `Partner information` to match the sentence case of other columns